src/nvim/fileio.c

```diff
@@ -707,7 +707,7 @@ readfile (
      * always using the GUI.
      */
     if (read_stdin) {
-      mch_msg(_("Nvim: Reading from stdin...\n"));
+      mch_msg(_("Nvim: Reading from stdin ...\n"));
```

Comment:
The problem with changing translated messages (wrapped in `_("...")`) is that all the .po files must be updated too :(